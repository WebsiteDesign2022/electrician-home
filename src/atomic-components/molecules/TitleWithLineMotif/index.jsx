import React from 'react';
import TopLeftTriangle from '../../atoms/TopLeftTriangle';

const TitleWithLineMotif = ({ title }) => (
    <div className="border-bottom align-content-start border-dark align-content-start col-3">
        <TopLeftTriangle color="black"/>
        <h3 align="left">{title}</h3>
    </div>    
);

export default TitleWithLineMotif;