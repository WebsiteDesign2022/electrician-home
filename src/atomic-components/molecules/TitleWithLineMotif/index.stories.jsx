import React from 'react';

import TitleWithLineMotif from '.';

export default {
  title: 'Electrician-Home/molecules/TitleWithLineMotif',
  component: TitleWithLineMotif,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <TitleWithLineMotif {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {
    title: 'A Title with Line Motif'
};
