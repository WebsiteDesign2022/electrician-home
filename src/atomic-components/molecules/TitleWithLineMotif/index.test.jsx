import TestRenderer from 'react-test-renderer';

import TitleWithLineMotif from '.';

describe("TitleWithLineMotif", () => {
    test("renders TitleWithLineMotif", () => {
        const titleWithLineMotif = TestRenderer.create(
            <TitleWithLineMotif/>
        );

        expect(titleWithLineMotif.toJSON()).toMatchSnapshot();
    });
});
