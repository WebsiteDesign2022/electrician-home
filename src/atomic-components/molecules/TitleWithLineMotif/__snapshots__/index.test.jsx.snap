// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleWithLineMotif renders TitleWithLineMotif 1`] = `
<div
  className="border-bottom align-content-start border-dark align-content-start col-3"
>
  <div
    className="arrow-top-left-black"
  />
  <h3
    align="left"
  />
</div>
`;
